fix-delete-userphoto

miyabitanimchi opened the pull request:
Added conditionals so the deleting photo functionality works only when the user changed the user photo.
Please check when you have time, thank you so much.

BlackStone8960 reviewed (commented):
Thank you so much, but can you just make it to delete user's photo in database after they push SAVE button in profile page?

miyabitanimchi commented:
I removed the conditional on line 124 on ProfileForm.js. So now the function goes to find the storage and tries to delete a user photo no matter what. I tested and deleted an account without setting the photo 3 times, but the performance was still ok.

BlackStone8960 reviewed (approved):
Looks good, good job!
